spec/game_spec.rb

```diff
@@ -31,12 +31,13 @@
 
   it "can create Game objects using the create_from_csv method" do
     new_game = Game.create_from_csv("./data/games_dummy.csv")
-    expect(@game1.game_id).to eq 2012030221
-    expect(@game1.season).to eq 20122013
-    expect(@game1.type).to eq "Postseason"
-    expect(@game1.away_team_id).to eq 3
-    expect(@game1.home_team_id).to eq 6
-    expect(@game1.away_goals).to eq 2
-    expect(@game1.home_goals).to eq 3
+    starter = new_game.first
```

Comment:
Good catch. This was previously not testing anything new, so this is much better.